api/v1/models/AttendeeExtraInfo.js

```diff
@@ -1,28 +1,18 @@
 const Model = require('./Model');
 const AttendeeExtraInfo = Model.extend({
-  tableName: 'attendee_extra_infos',
+  tableName: 'attendee_extra_info',
   idAttribute: 'id',
   validations: {
     attendeeId: ['required', 'integer'],
-    info: ['required', 'string', 'maxLength:255']
+    website: ['required', 'string', 'maxLength:255']
```

Comment:
lets leave this as info so that this field can be used as a generic field for anything else the user might want to add (the registration form suggests that they add additional supporting websites)

Reply:
Fixed 514a0cf64d43703da2c4107c3baa755bf371822f